glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -333,6 +333,17 @@ definitions:
             - description
         default: {}
 
+      gecko_datapoint:
+        title: Gecko Datapoint
+        description: |
+          This is a Gecko-specific property. It is the name of the Gecko histogram to
+          accumulate the data from, when using the Glean SDK in a product using GeckoView.
+          See bug 1566356 for more context.
+
+          This is valid when `type`_ is `timing_distribution`.
```

Comment:
This new parameter also needs an entry in `docs/metrics-yaml.rst` so it will show up in the docs.